### ISSUE TITLE
Cannot create database with 2.1.7 release

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#5464]
Hello, I download http://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.1.7.tar.gz&os=linux and try to create a database but I got a server error. A database with the name `null` is created!

with 2.1.6 http://orientdb.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.1.6.tar.gz&os=linux it works.

Perhaps this is helpful:
```sql

2015-12-14 23:28:20:826 WARNI Not enough physical memory available for DISKCACHE: 2,001MB (heap=494MB). Set lower Maximum Heap (-Xmx setting on JVM) and restart OrientDB. Now running with DISKCACHE=256MB [orientechnologies]
2015-12-14 23:28:20:833 SEVER Can not change cache size in runtime because storage engine plocal was not registered [OGlobalConfiguration$1]
```

**Comment by maggiolo00** [comment]
@StarpTech 

which exception did you get?

**Comment by maseev** [comment]
by the way, you can create a database through the console.sh by issuing something like

```
CREATE DATABASE remote:localhost/db-name root root plocal
```

but yeah, it's very sad that such a essential thing has been broken.

**Comment by smolinari** [comment]
I can confirm adding a DB to 2.1.7 through Studio doesn't work either. 

Also see: https://groups.google.com/forum/#!topic/orient-database/vpaYj_-5Kq4

Unfortunately, I can't test 2.1.6 to double check to make sure the problem isn't with my setup, but will this evening.

Scott

**Comment by maseev** [comment]
Here's what I get when I'm trying to create a database through the Orient Studio:

```
2015-12-15 14:19:35:935 INFO  {db=null} Creating database plocal:/Users/mironaseev/Documents/orientdb-community-2.1.7/databases/null [OServerCommandPostDatabase]
2015-12-15 14:19:36:578 WARNI ->0:0:0:0:0:0:0:1: Command not found: GET.sqlProfiler/null [ONetworkProtocolHttpDb]
```

When I click on the create database button the client sends to the server a POST request to the following URL:

```
http://localhost:2480/database/null/plocal/graph
```

Even though I specified the database name (in my case it was test), the client sent null instead of test.
It seems that the problem on the client side (Orient Studio). It for some reasons sends null instead of the database name that user specifies in the form.

**Comment by smolinari** [comment]
Actually, I just tested 2.1.6 (changing environments is relatively easy with spider-box) and it works fine.

Scott

**Comment by StarpTech** [comment]
@smolinari yes 2.1.6 works look at the issue description but 2.1.7 fails.

**Comment by smolinari** [comment]
@StarpTech - please read my first reply to this issue, then you'll better understand my second reply. 

Scott

**Comment by luigidellaquila** [comment]
Hi guys

sorry for this inconvenience, it's a problem in the packaging of Studio plugin.
We are working to quickly release a new hotfix.

In the meantime, could you please try to replace the Studio plugin (in the /plugins directory) with this?

https://github.com/orientechnologies/orientdb/blob/2.1.x/server/plugins/studio-2.1.zip?raw=true

Thanks

Luigi

**Comment by maseev** [comment]
Confirm. It works after I replaced old plugin with the one you provided @luigidellaquila. Now I can create a database with Orient Studio.

**Comment by StarpTech** [comment]
It is still an issue. I can't create a db in studio from the offical community/enterprise editions but with your seperate Studio plugin build I can fixed it.

**Comment by lvca** [comment]
This has been fixed by @maggiolo00 in 2.1.x branch. I'm going to release 2.1.8...
